instrument/call.go

```diff
@@ -48,20 +48,20 @@ func NewCall(scope tally.Scope, name string) Call {
 type call struct {
 	scope   tally.Scope
 	success tally.Counter
-	error   tally.Counter
+	err     tally.Counter
 	timing  tally.Timer
 }
 
 func (c *call) Exec(f ExecFn) error {
 	sw := c.timing.Start()
+	err := f()
```

Comment:
Would it be worth providing separate optional timing stats for success latencies and error latencies in addition to the overall timing stats? The overall timing stat may be insufficient if the user wants to gain additionally visibility into e.g., error latencies since that can be very different from the overall latency.

Reply:
If we separate the latencies based on success or error I think we would probably want to include a timer for both as well. Otherwise one wouldn't be able to calculate the true latency across all requests.